src/lennybot/check/docker_image_available.py

```diff
@@ -4,10 +4,11 @@
 
 import requests
 
-from ..config.config import LennyBotCheckConfig
+from ..config.config import LennyBotCheckConfig, LennyBotConfigContainerConfig, LennyBotConfigContainerRegistry
 from .icheck import ICheck
 
 PATTERN = r"(?:([\-\_\.\w]+)$)|(?:([\-\_\.\w]+)/([\-\_\.\w]+)$)|(?:([\-\.A-z0-9]+)/([\-\_\.\w]+)/([\-\_\.\w]+)$)"
+BLOCKED_REGISTRIES = []  # "docker.elastic.co"]  # Requires authentication, but is a public registry
 
```

Comment:
Create the `url_encode_params` function which takes a dictonary and returns an url encoded string of the parameters.